packages/subgraph/scripts/generate-manifests.ts

```diff
@@ -0,0 +1,25 @@
+import yaml = require('js-yaml');
+
+import Handlebars = require('handlebars');
```

Comment:
THAT'S the lib name I was trying to remember a couple days ago 😆 @mendesfabio

Reply:
why did you need it? 😅

Reply:
dunno 🤫 think I was trying to show @luizakp something